packages/ui/src/components/List/style.less

```diff
@@ -14,6 +15,7 @@
     align-items: end;
     width: 100%;
     border-bottom: none;
+    padding: 6px 0;
```

Comment:
Strange value....not 4px * x, and not % of em, what's the source of it

Reply:
That's because padding is a cumulative value. And total padding of `li` will give a 12px from top and bottom

Reply:
then it probaly should be something like :not(:last-of-type) { padding-bottom: 12px }